src/functions.cpp

```diff
@@ -1,26 +1,30 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"  // TODO: add proper compilation flag and change to #include "functions.hpp"
+
 #include <algorithm>
 #include <iostream>
 #include <limits>
 
-void print(std::vector<int> const & numbers) {
+void print(std::vector<int> const& numbers) {
     for (int i = 0; i < numbers.size(); ++i) {
```

Comment:
use std::size_t

Reply:
I didn't change anything, it was int from the beginning

Reply:
it is a part of homework, add alert flags first then watch what happend

Reply:
Yup, try to compile this code now with `make`. It should not compile.

Reply:
Right, my bad. It should work now. Thanks! :D